oak-run-commons/src/main/java/org/apache/jackrabbit/oak/run/PurgeOldIndexVersionCommand.java

```diff
@@ -28,11 +29,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import joptsimple.OptionParser;
-import joptsimple.OptionSet;
-import joptsimple.OptionSpec;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
```

Comment:
Could you please revert the unintended changes in file imports.

Reply:
is there anything apart from the order changes that you see problematic ?